packages/reactivity/src/ref.ts

```diff
@@ -3,8 +3,10 @@ import { OperationTypes } from './operations'
 import { isObject } from '@vue/shared'
 import { reactive } from './reactive'
 
+export const refSymbol = Symbol()
+
 export interface Ref<T> {
-  _isRef: true
+  _isRef: symbol
```

Comment:
Typo?
```suggestion
  _isRef: refSymbol
```

Reply:
No, shouldn’t use a implementation to a type

Reply:
Shouldn't it be `Symbol` then or it doesn't matter?